sites/platform/src/create-apps/app-reference.md

```diff
@@ -375,31 +377,61 @@ applications:
 ```
 {{% /version/specific %}}
 
-For examples of how to set up a `service` mount, see the dedicated [Network Storage page](/add-services/network-storage.md).
-
 The accessibility to the web of a mounted directory depends on the [`web.locations` configuration](#web).
 Files can be all public, all private, or with different rules for different paths and file types.
 
 Note that when you back up an environment,
 the mounts on that environment are backed up too.
 
 Also, mounted directories aren't deleted when they're removed from `{{< vendor/configfile "app" >}}`.
-The files still exist on disk until manually removed (or, for `tmp` mounts, until the app container is moved to another host during a maintenance operation).
+The files still exist on disk until manually removed.
+Or, for `tmp` mounts, **until the app container is moved to another host during a maintenance operation.**
 
 {{% version/only "2" %}}
 
-### Mounts, instances, and Network Storage
+### Data sharing through mounts
 
-`storage` mounts can be shared **between different instances of the same app**,
+By default, [`storage` mounts](#mounts) can be shared **between different instances of the same app**,
 which enables [horizontal scaling](/manage-resources/_index.md).
 
-In a [multi-application context](/create-apps/multi-app/_index.md), if you want data in a mount to be shared **between applications**,
-you must explicitly define and use a [Network Storage](/add-services/network-storage.md) service.
-Note that workers share the Network Storage instance of their parent app container.
+In a [multi-application context](/create-apps/multi-app/_index.md),
+you can even share a `storage` mount **between different applications** in the same project.
+
+For example, say you have a `backend` app and a `frontend` app.
+You want both apps to share data from the same mount.</br>
+Use the following configuration:
+
+```yaml {configFile="app"}
+# Define a storage mount in your backend app's configuration.
+backend:
+  mounts:
+    # The directory in the app container where you want your mount to be (mount path).
+    var/uploads:
+      # The mount type.
+      source: storage
+      # The subdirectory within the mounted disk (the source) where the mount points.
+      source_path: backend/uploads
+
+# Give your frontend app access to the mount defined in your backend app's configuration.
+frontend:
+  mounts:
+    # The directory in the app container where you want your mount to be (mount path).
+    web/uploads:
+      # The mount type.
+      source: storage
+      # Grants access to the backend app's mount. 
+      service: backend
+      # The remote source_path in the backend app's mount to enable data sharing.
+      source_path: backend/uploads
+```
+
```

Comment:
I think we should write one or two sentences explaining what the snippet does. It might not be totally self-explanatory